xml/System.Net/IPAddress.xml

```diff
@@ -2085,12 +2085,10 @@
         <ReturnType>System.Net.IPAddress</ReturnType>
       </ReturnValue>
       <Parameters>
-        <Parameter Name="ipString" Type="System.ReadOnlySpan&lt;System.Char&gt;" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1;netcore-3.1" />
```

Comment:
I think we need to keep this for .NET Core 2.1 - 3.1.

Reply:
But that belongs to the other overload, just above this one:
```
<Member MemberName="Parse">
      <MemberSignature Language="C#" Value="public static System.Net.IPAddress Parse (string ipString);" />
```

This overload has only the `ipSpan` parameter. Or is there some interdependence between the overloads docs? How does that work?

Reply:
It looks like the parameter was renamed in .NET 5. The XML keeps both versions of the method signature. You can see the old one here:

<img width="649" alt="image" src="https://github.com/dotnet/dotnet-api-docs/assets/24882762/5fe70265-fcdb-4d7b-9fdf-75bbb88a7ad5">